cpp/include/Ice/Communicator.h

```diff
@@ -357,7 +357,7 @@ namespace Ice
          * @return A proxy to the main ("") facet of the Admin object, or nullopt if no Admin object is configured.
          * @see #createAdmin
          */
-        std::optional<ObjectPrx> getAdmin() const; // NOLINT:modernize-use-nodiscard
+        std::optional<ObjectPrx> getAdmin() const; // NOLINT(modernize-use-nodiscard)
```

Comment:
As I understand it, :xxx is a comment on why we suppressed it, not the rule that is suppressed.